src/Uno.UI/UI/Xaml/XamlFilePathHelper.shared.cs

```diff
@@ -27,9 +27,10 @@ internal static string ResolveAbsoluteSource(string origin, string relativeTarge
 				var trimmedPath = relativeTargetPath.TrimStart(AppXIdentifier);
 				return trimmedPath;
 			}
-
-			// Strip leading forward-slash, if any, to match the path the target file is indexed by
-			relativeTargetPath = relativeTargetPath.TrimStart(new[] { '/' });
```

Comment:
@davidjohnoliver The current behavior was introduced for https://github.com/unoplatform/uno/issues/3361
The issue claiming that leading `/` shouldn't affect anything, but that seems incorrect to me.

cc @xperiandri

Reply:
Looks good, but can you add a comment explaining what's happening here (ie the '/' means to treat the path as an absolute path, so we return it without trying to resolve the relative path)